cmd/minishift/cmd/addon/addon_install_test.go

```diff
@@ -58,31 +60,23 @@ func Test_install_with_enable_flag_works(t *testing.T) {
 	defer cli.TearDown(tmpMinishiftHomeDir, tee)
 
 	addOnManager := GetAddOnManager()
-	if len(addOnManager.List()) != 0 {
-		t.Fatal(fmt.Sprintf("There should be no add-ons installed. Got got : %v", addOnManager.List()))
-	}
+
+	assert.Len(t, addOnManager.List(), 0)
```

Comment:
Again, we are losing the message here. Really much in all cases the assert function variant with the additional message should be used.

Reply:
Here we are loosing "There should be no add-ons installed." for example. It can get really hard to match a generic assertion failure to its occurrence w/o a custom message.

Reply:
assert.Len() gives a message like this on failure by default:
```
--- FAIL: Test_install_with_enable_flag_works (0.00s)
	assertions.go:241: 
                        
	Error Trace:	addon_install_test.go:64
		
	Error:		"[]" should have 2 item(s), but has 0
```

Reply:
> Here we are loosing "There should be no add-ons installed." for example. It can get really hard to match a generic assertion failure to its occurrence w/o a custom message.

+1 yes we need custom message here. but i don't think we need them for all the assertions.